### PR TITLE
Test coverage for the Sources module

### PR DESCRIPTION
**Pull request by jmanero-r7**

### CODE REVIEW

File: lib/sources/iindex.js [hunk 1]
```diff
@@ -24,7 +30,7 @@ class Index {
         config.name = config.type + ':' + // eslint-disable-line no-param-reassign
           Crypto.randomBytes(8).toString('hex'); // eslint-disable-line rapid7/static-magic-numbers
 
-        Log.log('WRAN', `Source configuration does not have a \`name\` parameter! Generated ${name}`);
+        Log.log('WRAN', `Source configuration does not have a \`name\` parameter! Generated ${config.name}`);
```
Comment (dgreene-r7) [inline lib/sources/iindex.js]: Typo

**Comment by dgreene-r7** [comment]
I noticed that you have a lot of disabled eslint rules at the top of your test files. You can put them in a `.eslintrc` file in the `test/` directory and disable the rules that way. Might make it easier, I dunno.

**Comment by fmitchell-r7** [comment]
Thanks for getting the test coverage up to 97%!

Other than the "WRAN" vs. "WARN" spelling mistake @dgreene-r7 pointed out, this looks good to me.

**Comment by fmitchell-r7** [comment]
👍  :shipit:

**Pull request merged**